custom-steps/ffmpeg/post-build.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
```

Comment:
Would it be possible to make this also `Import-Module "$PSScriptRoot/../../ps-modules/MacUtil"` if we're on MacOS? So calling code only needs to import one thing?

Reply:
Sure thing!

Reply:
These are now organized thusly: https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/1/commits/95e76ace903d6834cb63456c5e93f2f834530293